src/components/router/Router.js

```diff
@@ -0,0 +1,27 @@
+import { renderMainPage } from '../../views/MainPage/mainPage.js';
+import { settings } from '../../main.js';
+
+const app = document.getElementById('app');
+
+const applicationRoutes = {
+  '/': renderMainPage,
+  '#quiz-settings': settings,
+};
+
+function checkRoute(path) {
+  if (applicationRoutes[path] === undefined) {
+    return () => alert('Podana strona nie istnieje!');
+  } else {
+    return applicationRoutes[path];
+  }
+}
```

Comment:
this may be confusing - a better name would be setRenderFunction (it will return render function based on the path)